Terminal.Gui/Configuration/SourceGenerationContext.cs

```diff
@@ -20,5 +20,6 @@ namespace Terminal.Gui;
 [JsonSerializable (typeof (Dictionary<ColorName16, string>))]
 [JsonSerializable (typeof (Dictionary<string, ThemeScope>))]
 [JsonSerializable (typeof (Dictionary<string, ColorScheme>))]
+[JsonSerializable (typeof (Rune))]
```

Comment:
I think you don't need this here because what you are serialize/deserialize are the `GlyphDefinitions` class and the `[JsonSerializable (typeof (GlyphDefinitions))]` is already included.

Reply:
`Key.Separator` is a Rune.

Reply:
> `Key.Separator` is a Rune.

I know but there was already added `[JsonSerializable (typeof (Key))]` in the `SourceGenerationContext` class, which mean that will use `Key` for serialize/deserialize and the `RuneJsonConverter` to the conversion. Try without the `[JsonSerializable (typeof (Rune))]` and see what happens. If it fail then add it again.